make/deps.mk

```diff
@@ -52,7 +52,7 @@ ifeq ($(CIRCLECI),true)
     DEPS += proper
 endif
 
-dep_ranch = git https://github.com/2600hz/erlang-ranch 1.7.1
+dep_ranch = git https://github.com/ninenines/ranch.git 1.8.x
```

Comment:
As we spoke about on the phone this should be pinned to 1.8.0 tag, not to the 1.8.x branch